packages/block-library/src/query/edit/inspector-controls/enhanced-pagination-control.js

```diff
@@ -25,19 +25,19 @@ export default function EnhancedPaginationControl( {
 		);
 	} else if ( enhancedPagination ) {
 		help = __(
-			"Browsing between pages won't require a full page reload, unless non-compatible blocks are detected."
+			'Reload the full page—instead of just the posts list—when visitors navigate between pages.'
 		);
 	} else if ( hasUnsupportedBlocks ) {
 		help = __(
-			"Force page reload can't be disabled because there are non-compatible blocks inside the Query block."
+			'Enhancement disabled because there are non-compatible blocks inside the Query block.'
```

Comment:
The word 'enhancement' is still biased as it assumes one of the two navigation model is, in a way., 'better'. As I mentioned in https://github.com/WordPress/gutenberg/issues/63599#issuecomment-2250167186, to me WordPress should not make any assumption on what kind of experience is 'better' for users. It should present the feature in a neutral way,

Reply:
Agreed on not using words like "enhancement" which are going to be contextual and instead describing how things work in a way people can understand.